Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -21,6 +23,19 @@ open class ChatChannelListItemView<ExtraData: ExtraDataTypes>: View, UIConfigPro
     }
     
     // MARK: - Subviews
+
+    public private(set) lazy var actionButtonStack: UIStackView = {
+        let button = UIButton().withoutAutoresizingMaskConstraints
+        button.setImage(UIImage(named: "icn_delete", in: .streamChatUI), for: .normal)
+        button.backgroundColor = uiConfig.colorPalette.channelListActionsBackgroundColor
+        button.tintColor = uiConfig.colorPalette.channelListActionDeleteChannel
```

Comment:
The `defaultAppearance` looks the right place for the styling

Reply:
Thanks, fixed with new superclass :)